<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotDialog</class>
 <widget class="QDialog" name="ScreenshotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Take Screenshot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 12px &quot;Noto Sans&quot;, &quot;FreeSans&quot;;
	background: #1A1A1A;
	color: whitesmoke;
}

/* QCheckBox */

QCheckBox::indicator,
QGroupBox::indicator{
	width: 11px;
	height: 11px;
	border: 1px solid #008098;
	border-radius: 2px;
}
QCheckBox::indicator:unchecked,
QGroupBox::indicator:unchecked {
	border: 1px solid whitesmoke;
	background: #1A1A1A;
	image: none;
}
QCheckBox::indicator:checked,
QGroupBox::indicator:checked {
	border: 1px solid #008098;
	background: #008098;
	image: url(:/img/checked.svg);
}

/* QComboBox */

QComboBox {
	background: #454A4D;
	border: 1px solid black;
	border-radius: 2px;
	padding: 2px;
}
QComboBox:focus {
	border: 1px solid #008098;
}
QComboBox::drop-down {
	background: #008098;
	border-top-right-radius: 2px;
	border-bottom-right-radius: 2px;
}
QComboBox::down-arrow {
	image: url(:/img/down-arrow.svg);
}

/* QLineEdit */

QLineEdit {
	background: black;
	border: 1px solid #454A4D;
	border-radius: 2px;
	padding: 2px;
}
QLineEdit:focus {
	border: 1px solid #008098;
}

/* QPushButton */

QPushButton {
	background: #454A4D;
	border: 1px solid #454A4D;
	border-bottom: 1px solid black;
	border-radius: 2px;
	color: whitesmoke;
	min-width: 76px;
	min-height: 17px;
	padding: 4px 6px;
}
QPushButton:focus {
	border: 1px solid #008098;
	outline: none;
}
QPushButton:pressed {
	background: #2E3133;
}
QPushButton:disabled {
	background: transparent;
	border: 1px solid #454A4D;
	color: #B8B8B8;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="browseButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    min-width: 40px;
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="locationEdit">
       <property name="placeholderText">
        <string>Save Location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="templateEdit">
       <property name="placeholderText">
        <string>File Name Template</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="formatComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tga</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="subtitlesCheckBox">
       <property name="text">
        <string>Sho&amp;w subtitles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showCheckBox">
       <property name="text">
        <string>Always show this dialog</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>locationEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>templateEdit</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>subtitlesCheckBox</tabstop>
  <tabstop>showCheckBox</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
